tables/tests/test_tables.py

```diff
@@ -993,18 +993,30 @@ def test02b_AppendRows(self):
 
             row.append()
             # the next call can mislead the counters
+            #table.flush()
```

Comment:
Can you add a comment here that we are closing and reopening in 'r'ead-only for making windows happy and make it use the optimized path for reading Blosc2 chunks?